client/src/components/Community/Community.css

```diff
@@ -1,21 +1,218 @@
-.community-component
-{
-     margin-top:100px;
-}
-.Society-cards .heading
-{
-  text-align: center;
-  color: #ffffff;
-  font-style: italic;
-  font-weight: 200;
-}
-.Society-cards .carousel-indicators
-{
-  display: none;;
-}
-.Society-cards .card
-{
-  margin:40px auto;
-  border-radius: 5px;
-  min-height: 20rem;;
+.community-component {
+  color: white;
+  font-family: 'Raleway';
 }
+
+.community-component .union {
+  height: 0;
+  pointer-events: none;
+}
+
+.community-component .union-img {
+  height: 250vh;
+  width: auto;
+  position: relative;
+  left: 48vw;
+  bottom: 58vh;
+}
+
+.community-component .main {
+  margin-top: 35vh;
+}
+
+.community-component .side-banner {
+  color: #1EABD6;
+  height: 0;
+  width: 30vw;
+  position: absolute;
+  left: 82vw;
+  font-size: 1.5vw;
+  transform: rotate(-90deg);
+}
+
+.community-component .title {
+  opacity: 0;
+  width: 0;
+  height: 0;
+  pointer-events: none;
+  margin:0;
+  padding: 0;
+}
+
+.community-component .community-card {
+  border: 1px solid #EE2A7B;
+  box-sizing: border-box;
+  border-radius: 23px;
+  height: 45vh;
+  margin-left: 1vw;
+  margin-bottom: 8vh;
+  position: relative;
+  background-position: center;
+  transition: background-size 1s;
+}
+
+.community-component .link {
+  text-decoration: none;
+  font-family: 'Raleway';
+  color: #1EABD6;
+}
+
+.community-component .community-card .name {
+  font-size: 3.8vh;
+  font-weight: bold;
+  text-shadow: #EE2A7B 1px;
+}
+
+.community-component .community-card .small-name {
+  height: 0;
+  width: 0;
+  pointer-events: none;
+  opacity: 0;
+}
+
+.community-component .community-card .large-name {
+  transform: translate(0%, 300%);
+}
+
+.community-component .competitive-coding {
+  background: linear-gradient(108.96deg, rgba(238, 42, 123, 0.45) 0%, rgba(0, 0, 24, 0.675) 43.75%), 
+  url(./competitive-coding.jpg);
+  background-size: 104%;
```

Comment:
you can add `background-size : cover` and can apply the same to all the images.

Reply:
I did that but the hover animation wasn't happy after that. The animation would go from `cover` to `150%` which doesn't yield the desired animation. Instead it jumps after the animation duration.

Reply:
Done.